src/index.ts

```diff
@@ -12,10 +13,11 @@ export type UtopiaTypeConfig = {
   negativeSteps?: number;
   positiveSteps?: number;
   relativeTo?: UtopiaRelativeTo;
+  stepLabelsStyle?: UtopiaStepLabelStyle
 }
 
 export type UtopiaStep = {
-  step: number;
+  step: string | number;
```

Comment:
I wonder if adding a `label` property would be a neater move, rather than changing a property on the existing API?

Reply:
Thanks for the suggestion. Yes, adding the `label` property is probably the more sensible default here. Will make the change.